CiscoMeraki/ModelingRules/CiscoMerakiModelingRules/CiscoMerakiModelingRules_schema.json

```diff
@@ -0,0 +1,7 @@
+{
+    "Cisco_Meraki_raw": {
+      "_raw_log": {
+        "type": "string",
+        "is_array": false
+      },
+  }
```

Comment:
Missing bracket, the JSON format is invalid

Reply:
We compared it to cisco SMA PR and did not see a  bracket, please correct me if I am wrong. 
{
    "Cisco_SMA_raw": {
      "_raw_log": {
        "type": "string",
        "is_array": false
      },
  }